ang/civicase/CaseListTable.js

```diff
@@ -155,6 +155,10 @@
       var contacts = [];
 
       _.each(cases, function (data) {
+        contacts = contacts.concat(CRM._.map(data.contacts, function (contact) {
```

Comment:
Are we testing this logic ?

Reply:
`CaseListTable.js` is a big file. So the unit tests are not added as it will increase the scope of the ticket by a lot.